src/routes/safe/components/Apps/components/AppsList.test.tsx

```diff
@@ -30,6 +32,9 @@ jest.mock('@gnosis.pm/safe-react-gateway-sdk', () => ({
         fetchStatus: 'SUCCESS',
         chainIds: ['1', '4'],
         provider: null,
+        accessControl: {
+          type: 'NO_RESTRICTIONS',
```

Comment:
Do you not want to use the enum `accessControl.type`s for each mocked object here?

Reply:
I do not see a strong argument for either approach. My thoughts were is that I describe the JSON we expect in a plain format without much implementation details. I'm ok with changing it if you insist